Wikipedia/Code/WebViewController.m

```diff
@@ -296,6 +296,12 @@ - (WMFArticleFooterView*)footerLicenseView {
 }
 
 - (void)addFooterView {
+    if(!self.article){
+        return;
+    }
+    if([self.article.title isNonStandardTitle]){
```

Comment:
Should this be happening in the same place where we check if an article is a main page before showing a footer?